components/photoslider.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { Box, Button, Container, HStack } from '@chakra-ui/react';
+
+function PhotoSlider() {
```

Comment:
This code looks good, but there's also a way we can use pre-built components from Swiper.js (makes transitions smoother between slides and other goodies). 

We should change this component to use swiper.js instead of from scratch: https://swiperjs.com/react

Reply:
fixed